### PR TITLE
Hiero: Initial frame publish support.

### PR DESCRIPTION
**Pull request by tokejepsen**
With the "frame" family tags, you can publish still images from the edit by tagging frames in the sequence.

We use this for outputting promotional stills at the end of the project.

### CODE REVIEW

File: pype/plugins/hiero/publish/collect_frames.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+from pyblish import api
+
+
+class CollectFrames(api.ContextPlugin):
+    """Collect frames from tags.
+
+    Tag is expected to have metadata:
+    {
+        "family": "frame"
```
Comment (jezscha) [inline pype/plugins/hiero/publish/collect_frames.py:13]: I would rather use family `image` so we don't need to create new family. Hiero is not using in context `image` family so you will still have the protection from the families blending.

**Review by jezscha — COMMENTED** [review]
I love the workflow of the images and I certainly see their use for Matepaint and the reference images. I am concerned about another creation of the family only if we could use the one we already have and which is not used in the Hiero.

File: pype/plugins/hiero/publish/collect_frames.py (hunk 1 above)
Comment (tokejepsen) [reply pype/plugins/hiero/publish/collect_frames.py:13]: When collecting the frame family gets the `image` family as well. That's how its integrated.

I thought `frame` was more intuitive in the context of NukeStudio where the user is interested in publishing a single frame.

Comment (mkolar) [reply pype/plugins/hiero/publish/collect_frames.py:13]: from what I can see it seem that the final main family will end up being `frame` I agree with @jezscha. It's absolutely ok to have helper families to make it more familiar to the user in their context, but the published main family should be image to keep it consistent.

This this what it visible in Loader at the end of the day, would be `frame`

Comment (tokejepsen) [reply pype/plugins/hiero/publish/collect_frames.py:13]: Ahh I see. Yeah, it'll be registered as `frame` atm in the loaders. Will amend.

**Review by jezscha — APPROVED** [review]
awesome, thanks!

**Review by mkolar — APPROVED** [review]

**Comment by mkolar** [comment]
Happy to merge now. will leave opened for a littler while just while we convert to 3.0 as well

**Comment by tokejepsen** [comment]
Sorry, just slipped in some improvements.

**Pull request merged**